### PR TITLE
Fix: user creation failure while cluster deployed with 4.x ver and rhbuild was 5.x

### PR DESCRIPTION
**Pull request by anrao19**
…ld was given 5.x

Signed-off-by: Anuchaithra Rao <anrao@redhat.com>

# Description
Latest LOG: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Z62A93/create_user_0.log

Fix for user creation failure when cluster was deployed with 4.x and rhbuild was 5.x
In user craetion decision was craeted using rhbuild
Log: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/create_user_0.log

cli: python3 run.py --rhbuild 5.2 --platform rhel-8 --global-conf conf/pacific/rgw/rgw_multisite.yaml --osp-cred <cred>.yaml --inventory conf/inventory/rhel-8-latest.yaml --suite suites/pacific/rgw/tier-1_rgw_multisite_test-upgrade-4-to-latest.yaml --log-level info --ignore-latest-container --instances-name 43_upgrade_52 --store --skip-sos-report --build tier-0 

And also included some polarion ids

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/rgw/sanity_rgw_multisite.py [hunk 1]
```diff
@@ -92,7 +92,10 @@ def run(**kw):
         set_test_env(config, primary_rgw_node)
         set_test_env(config, secondary_rgw_node)
 
-        if primary_cluster.rhcs_version.version[0] == 5:
+        ceph_version, rc = primary_rgw_node.exec_command(
```
Comment (udaysk23) [inline tests/rgw/sanity_rgw_multisite.py:95]: Can we replace `rc` with `_` if we are not using `rc` further

Comment (anrao19) [reply tests/rgw/sanity_rgw_multisite.py]: Made changes

File: tests/rgw/sanity_rgw_multisite.py [hunk 2]
```diff
@@ -92,7 +92,10 @@ def run(**kw):
         set_test_env(config, primary_rgw_node)
         set_test_env(config, secondary_rgw_node)
 
-        if primary_cluster.rhcs_version.version[0] == 5:
+        out, _ = primary_rgw_node.exec_command(
+            sudo=True, cmd="rpm -qa | grep ceph-common"
+        )
+        if int(out.split("ceph-common-")[-1].split(".")[0]) > 14 :
```
Comment (psathyan) [inline tests/rgw/sanity_rgw_multisite.py:98]: I believe you are adding support for RHCS 6.0 and I believe it is better to do
```
if primary_cluster.rhcs_version.version[0] >= 5:
```

Comment (anrao19) [reply tests/rgw/sanity_rgw_multisite.py]: Made changes as discussed

**Review by psathyan — APPROVED** [review]

**Pull request merged**